### PR TITLE
Remove notes on approval process

### PR DESCRIPTION
**Pull request by BanzaiMan**
See https://github.com/travis-ci/docs-travis-ci-com/pull/390

### CODE REVIEW

**Comment by meatballhat** [comment]
@BanzaiMan what's up?  Is this process being shut down? :sob:

File: README.md [hunk 1]
```diff
@@ -2,14 +2,4 @@
...
-0. Check the list of approved source for your build environment (most likely [`ubuntu`](./ubuntu.json)).
-0. If it's not in there, check for [existing issues requesting the source you 
-   want](https://github.com/travis-ci/apt-source-whitelist/issues), and if one doesn't exist please
-   open an issue requesting the source you need in the [this
-   repo](https://github.com/travis-ci/apt-source-whitelist/issues/new?title=APT+source+whitelist+request+for+___SOURCE___)
-   (and be sure to replace `__SOURCE__` in the issue title :wink:).  The body of the issue must include the "deb 
-   source line" which may be a ppa alias such as `ppa:fkrull/deadsnakes`.
-   If the deb source is not a PPA, you must also include the URL to the GPG key.
-0. Please be patient :smiley_cat:
+At this moment, we're unable to continue processing the APT sources requests. If you require a new source, please use `duo: required`in your `.travis.yml` file. Thank you!
```
Comment (BanzaiMan) [inline README.md]: Eschew contraction: "we're" → "we are"

Comment (BanzaiMan) [inline README.md]: It should be <tt>\`sudo: required\`</tt>. (Also, there should be a space after <tt>\`</tt>.)

Comment (BanzaiMan) [inline README.md]: Add documentation link: https://docs.travis-ci.com/user/installing-dependencies#Installing-Packages-with-the-APT-Addon.

**Pull request merged**